Gemfile

```diff
@@ -13,6 +13,7 @@ group :test, :development do
   gem 'simplecov'
   gem 'factory_bot_rails'
   gem 'rspec-collection_matchers', '~> 1.0'
+  gem 'rexml'
```

Comment:
Why do you need this? :)

Reply:
I needed it to run the specs. Maybe it’s because I used Ruby 3.0.1?

Reply:
```
Failure/Error:
  rescue Nokogiri::XML::SyntaxError, REXML::ParseException => e
    raise Saml::Errors::UnparseableMessage.new(e.message)
  rescue TypeError => e
    raise Saml::Errors::UnparseableMessage.new(e.message)
  rescue NoMethodError => e
    raise Saml::Errors::UnparseableMessage.new(e.message)

NameError:
  uninitialized constant Saml::Base::XmlMapperClassMethods::REXML
# ./lib/saml/base.rb:56:in `rescue in parse'
# ./lib/saml/base.rb:44:in `parse'
# ./lib/saml/provider_stores/file.rb:35:in `add_metadata'
# ./lib/saml/provider_stores/file.rb:29:in `block in load_files'
# ./lib/saml/provider_stores/file.rb:28:in `each'
# ./lib/saml/provider_stores/file.rb:28:in `load_files'
# ./lib/saml/provider_stores/file.rb:12:in `initialize'
# ./spec/support/saml_config.rb:2:in `new'
# ./spec/support/saml_config.rb:2:in `block in <top (required)>'
# ./lib/saml.rb:256:in `setup'
# ./spec/support/saml_config.rb:1:in `<top (required)>'
# ./spec/spec_helper.rb:24:in `require'
# ./spec/spec_helper.rb:24:in `block in <top (required)>'
# ./spec/spec_helper.rb:24:in `each'
# ./spec/spec_helper.rb:24:in `<top (required)>'
# ./spec/lib/saml/artifact_resolve_spec.rb:1:in `require'
# ./spec/lib/saml/artifact_resolve_spec.rb:1:in `<top (required)>'
# ------------------
# --- Caused by: ---
# LoadError:
#   cannot load such file -- rexml/document
#   /Users/chaimsolomon/.rvm/gems/ruby-3.0.1@libsaml/gems/activesupport-6.1.3.2/lib/active_support/xml_mini/rexml.rb:42:in `require'
You don't have rexml installed in your application. Please add it to your Gemfile and run bundle install
```

Reply:
Ahh ok, that explains indeed. I was using 2.6.6 locally. I will update the Travis file accordingly.